nerdlandbot/commands/pets.py

```diff
@@ -0,0 +1,309 @@
+import typing
+import discord
+from discord.ext import commands
+import asyncio
+import random
+from PIL import Image
+import io
+from pathlib import Path
+
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.commands.GuildData import get_guild_data, GuildData
+from nerdlandbot.helpers.constants import PETS_DIR_NAME, IMAGE_SIZE, INTERACT_TIMEOUT
+
+
+class Pets(commands.Cog, name="Pets"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(
+        name="add_pet",
+        brief="add_pet_brief",
+        help="add_pet_help",
+        usage="add_pet_usage",
+    )
+    @commands.guild_only()
+    async def add_pet(
+        self,
+        ctx: commands.Context,
+        name: typing.Optional[str] = None,
+        category: typing.Optional[str] = None,
+    ):
```

Comment:
```suggestion
    ) -> None:
```

I'm not sure what `ctx.send` returns, I'm assuming `None`. Though that would be the return type used here.